app.py

```diff
@@ -24,9 +26,10 @@
 all_sound_ids = json.load(open(PATH_TO_ALL_SOUND_IDS, 'rb')) 
 sound_id_user = json.load(open(PATH_TO_SOUND_IDS_PER_USER,'rb'))
 genres_file = json.load(open(PATH_TO_GENRE_FILE, 'rb'))
+joined_metadata = json.load(open(PATH_TO_JOINED_METADATA, 'rb'))
 
-default_N_assign_more_sounds = 5
-enable_auto_assign_annotations = False
+default_N_assign_more_sounds = 10
```

Comment:
maybe let's set it to 25?

Reply:
I prefer to set it for a low number that users can annotate just after asking for more. If we take 25 sounds and don't finish annotating them, the system will not know they have been attributed to someone already and, if anyone asks for more sounds in the meantime, there's the possibility of these sounds to be assigned twice.